_rules/aria-hidden-focus-6cfa84.md

```diff
@@ -4,33 +4,40 @@ name: aria-hidden with focusable content
 rule_type: atomic
 description: |
   This rule checks that elements with an `aria-hidden` attribute do not contain focusable elements
-success_criterion:
-  - 1.3.1 # Info and Relationships
-  - 4.1.2 # Name, Role, Value
-test_aspects:
+accessibility_requirements:
+  - wcag20: 1.3.1 
```

Comment:
@vemolista, can you keep the comments about the success criteria name ("# Info and Relationships" ) in the update?